clloader/scenario_loaders/classIncremental.py

```diff
@@ -0,0 +1,200 @@
+
```

Comment:
I like the idea of having different kind of loaders for different scenario.

Altought I'd prefer the sub-package to be called `scenario` instead of `scenario_loaders`, shorter is better (*i.e.* PyTorch api > TF api :p)
And also rename the module in snake case instead of camel case.

Reply:
And actually, the best would be to import the three scenario in the `__init__.py` of the `scenario` sub-package.

So the user would import likewise: `from clloader.scenario import ClassIncremental, InstanceIncremental`

I'm really fond of short import route ❤️

Reply:
snake case for module and camel case for classes?

Reply:
I recall that what PEP08 advise.